index.js

```diff
@@ -643,15 +643,13 @@ module.exports = {
    *
    * @param {string} table - the table to insert into
    * @param {Object} feature - a geojson feature
-   * @param {string} i - index value to use an id
    * @private
    */
-  _insertFeature: function (table, feature, i) {
+  _insertFeature: function (table, feature) {
     var featurestring = JSON.stringify(feature).replace(/'/g, '')
 
     if (feature.geometry && feature.geometry.coordinates && feature.geometry.coordinates.length) {
       var geohash = this.createGeohash(feature, this.geohashPrecision)
-      feature.geometry.crs = {'type': 'name', 'properties': {'name': 'EPSG:4326'}}
```

Comment:
so feature isn't even used after this point, right @kneemer? this is just a weird artifact?